# 获取参考文献



In [ ]:
* 路径设定

global path_PDF "../PDF"

cd $path_PDF



In [2]:
local doi 10.1016/j.jeconom.2025.106009
getiref `doi' , m dis

  
Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Economet
> rics, 250, 106009.
    Link    PDF    Google    

Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Economet
> rics, 250, 106009. [Link](https://doi.org/10.1016/j.jeconom.2025.106009), [PD
> F](http://sci-hub.ren/10.1016/j.jeconom.2025.106009), [Google](<https://schol
> ar.google.com/scholar?q=Pairwise valid instruments>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^
 
Sun_2025_Pairwise_valid_instruments


Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Econometrics, 250, 106009. [Link](https://doi.org/10.1016/j.jeconom.2025.106009), [PDF](http://sci-hub.ren/10.1016/j.jeconom.2025.106009), [Google](<https://scholar.google.com/scholar?q=Pairwise valid instruments>).